string[6] joint_names
float32[6] position
float32[6] velocity
batteryInfo[6] battery_level
float32[6] motor_current
geometry_msgs/Quaternion[6] imu